### DISCUSSION TITLE
中英文混用时英文会超出页边界

### DISCUSSION THREAD

**Discussion by Jamie-obj** [note286/xduts#107] [Q&A] [answered]
如图所示，请问这个问题该怎么解决呢
![image](https://user-images.githubusercontent.com/65389500/228158806-3ce30b5c-a1d4-47e9-a1fe-96d0422d5c4d.png)

**Comment by Jamie-obj** [comment]
我查到一种解决方案是使用`\linebreak`
https://github.com/TheNetAdmin/zjuthesis/issues/112#issue-775365353
或者使用`\hyphenation`
https://blog.csdn.net/z_feng12489/article/details/89964758

**Comment by note286** [comment, marked as answer]
你可以手动断词，例如`Deep\-Labvl`或者使用`\seqsplit`。

**Reply by Jamie-obj** [reply]
感谢感谢~

**Reply by note286** [reply]
原因就是你的这个词不在断词规则`C:\texlive\2023\texmf-dist\tex\generic\hyphen\hyphen.tex`里。

**Comment by sikouhjw** [comment]
参考 https://www.zhihu.com/question/583581138/answer/2891915263
